test/integration/order_integration_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class UserLoginTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+  attr_reader :user
+
+  def setup
+    @user = User.create(username: 'user',
+                      password: 'password',
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.